client/src/store/usePlayer.ts

```diff
@@ -23,3 +23,9 @@ export function useMe(): string {
   }
   return myName;
 }
+
+export function useCommonName(): [string | null, (key: string | null) => void] {
```

Comment:
common name is a weird name.  Maybe useLastUsedName?

Reply:
I'm fine with that